src/posthog-core.ts

```diff
@@ -989,8 +989,15 @@ export class PostHog {
             properties['$window_id'] = windowId
         }
 
-        if (this.sessionRecording) {
-            properties['$recording_status'] = this.sessionRecording.status
+        try {
+            if (this.sessionRecording) {
+                properties['$recording_status'] = this.sessionRecording.status
+                properties['$sdk_debug_replay_internal_buffer_length'] = this.sessionRecording['buffer'].data.length
+                properties['$sdk_debug_replay_internal_buffer_size'] = this.sessionRecording['buffer'].size
+            }
+            properties['$sdk_debug_retry_queue_size'] = this._retryQueue?.['queue']?.length
```

Comment:
this should give us a view on how much data replay is currently holding
and whether the SDK retry queue is high

hopefully giving signal that e.g. network requests to send data are failing in replay 
when smaller event data might be succeeding